manifests/native.json

```diff
@@ -248,6 +248,134 @@
       "mdnPath": "Global_Objects/Array/values",
       "category": "native"
     },
+    {
+      "type": "native",
+      "moduleName": "math.acosh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.acosh",
+      "mdnPath": "Global_Objects/Math/acosh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.asinh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.asinh",
+      "mdnPath": "Global_Objects/Math/asinh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.atanh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.atanh",
+      "mdnPath": "Global_Objects/Math/atanh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.cbrt",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.cbrt",
+      "mdnPath": "Global_Objects/Math/cbrt",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.clz32",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.clz32",
+      "mdnPath": "Global_Objects/Math/clz32",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.cosh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.cosh",
+      "mdnPath": "Global_Objects/Math/cosh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.expm1",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.expm1",
+      "mdnPath": "Global_Objects/Math/expm1",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.f16round",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.f16round",
+      "mdnPath": "Global_Objects/Math/f16round",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.fround",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.fround",
+      "mdnPath": "Global_Objects/Math/fround",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.hypot",
```

Comment:
this one too

Reply:
The same, https://github.com/es-shims/Math. hypot